### PR TITLE
some minor fix during test on ubuntu

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by hidekazuna** [comment]
@jichenjc I am catching up v1alpha2 Migration. Is this PR following patch for #446 ?

**Comment by jichenjc** [comment]
@hidekazuna yes, I am fixing a set of minor issues.

**Comment by hidekazuna** [comment]
@jichenjc Thanks, I will try this PR.

File: samples/templates/machines.yaml [hunk 1]
```diff
@@ -17,13 +17,14 @@ spec:
     apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
     namespace: default
     name: test-cluster-kube-master-01
-  version: "v1.15.0"
+  version: "1.15.0"
```
Comment (hidekazuna) [inline samples/templates/machines.yaml]: Could you update kubernetesVersion in L56 as well?

Comment (jichenjc) [reply samples/templates/machines.yaml]: @hidekazuna done

**Comment by sbueringer** [comment]
Sorry didn't saw this pr I'll rebase my new pr to these fixes

**Pull request merged**